python/ray/ml/examples/score_large_dataset.py

```diff
@@ -0,0 +1,63 @@
+#  Adapted from Highly Performant TensorFlow Batch Inference on Image Data
+#  Using the SageMaker Python SDK
+#  https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker_batch_transform/tensorflow_open-images_jpg/  # noqa
+import pandas as pd
+import tensorflow as tf
+
+import ray
+from ray.ml.batch_predictor import BatchPredictor
+from ray.ml.predictors.integrations.tensorflow import TensorflowPredictor
+from ray.ml.preprocessors import BatchMapper
+from ray.ml.train.integrations.tensorflow.tensorflow_trainer import TensorflowTrainer
+from ray import train
+
+
+# Should the logic here be consolidated into some preprocessor or DataSource to reuse?
+def per_element_transform(x: bytes) -> tf.Tensor:
+    image_tensor = tf.io.decode_image(x)
+    resized_image_tensor = tf.image.resize(image_tensor, [224, 224])
```

Comment:
this should probably not live here...

Reply:
it's a little weird that we are converting from a tensorflow tensor to numpy array.

should be in numpy/pandas land and you do your last mile preprocessing (in tensor format) --> directly to your model. Last last mile...